test/it/ArrivalSpec.scala

```diff
@@ -32,21 +33,23 @@ class ArrivalSpec extends IntegrationSpec {
 
   "Consignment References Page" when {
     "GET" should {
-      "return 200" in {
+      "throw feature disabled" in {
```

Comment:
I think for this test case we need to manually build `AppConfig` to always have a `ileQuery` enabled, because when someone will disable this locally and run tests it will fail, because there is no exception.

But I'm not sure if it's a mistake, because I can't imagine a situation when we will need to disable this locally, but it's somehow possible

Reply:
IleQuery is enabled by default in `IntegrationSpec` base class.

Reply:
In that case it's enough :)